Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -370,13 +370,13 @@ public class PBXProjGenerator {
 
     func generateExternalTargetDependency(from: String, to target: String, in project: String, platform: Platform) throws -> (PBXTargetDependency, Target, PBXReferenceProxy) {
         guard let projectReference = self.project.getProjectReference(project) else {
-            fatalError("project not found")
+            fatalError("project '\(project)' not found")
```

Comment:
Could you describe sample outputs?

I'm curious about how to express this object by `CustomStringConvertible` 👀

Reply:
Maybe you're misunderstanding.

`project` is String. Not object like class 👻